EIPS/eip-5375.md

```diff
@@ -0,0 +1,293 @@
+---
+eip: 5375
+title: NFT Authorship
+description: An extension of EIP-721 for NFT authorship and author consent.
+author: Samuele Marro (@samuelemarro), Luca Donno (@lucadonnoh)
+discussions-to: <URL>
```

Comment:
Must be a url on ethereum magicians.

Reply:
Fixed in 705c268b2c8488b5378c8a5b9492a359bd397c80.